tests/jobs/test_base_job.py

```diff
@@ -31,113 +31,115 @@
 from tests.test_utils.config import conf_vars
 
 
-class TestBaseJob(unittest.TestCase):
-    class TestJob(BaseJob):
-        __mapper_args__ = {
-            'polymorphic_identity': 'TestJob'
-        }
+class TestJob(BaseJob):
```

Comment:
Should we use something like `MockBaseJob`? I think `Test` prefix will make pytest to look for test methods.